DQM/RPCMonitorClient/interface/RPCOccupancyTest.h

```diff
@@ -36,11 +34,16 @@ class RPCOccupancyTest:public RPCClient {
   std::vector<MonitorElement *>  myOccupancyMe_;
   std::vector<RPCDetId>   myDetIds_;
   int prescaleFactor_;
- 
+
+  float totalActive_, totalStrips_;
```

Comment:
@acimmino - why floats? these are being used as counters, so an unsigned int would be the usual choice.